Source/Core/DolphinWX/Frame.h

```diff
@@ -21,6 +21,7 @@
 #include "Common/Event.h"
 #include "Core/ConfigManager.h"
 #include "DolphinWX/Globals.h"
+#include "LuaScripting.h"
```

Comment:
This can be a forward declaration.

Reply:
Doing this meant I had to move the include to FrameTools.cpp, since there's code in that file that actually accesses members of LuaScriptFrame.

Reply:
That's the point of a forward declaration. The dependency on the header is moved into a cpp file, so every header that includes this one, doesn't also include the Lua scripting header, reducing compile time and lessening the amount of identifiers available where they don't need to be.

Reply:
To get a basic listing of cases where forward declarations can be used instead of inclusions, there's also [this](https://stackoverflow.com/a/553869) answer on SO.

Reply:
Thank you for the resource, sorry for not knowing about this earlier.